### PR TITLE
DevWs for CoolKit Technology Co.,Ltd containing containing SmartSense Temp/Humidity Sensor

### PR DESCRIPTION
**Pull request by mingwei0827**

### CODE REVIEW

File: devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy [hunk 1]
```diff
@@ -177,12 +178,10 @@ def refresh() {
...
 			zigbee.readAttribute(zigbee.RELATIVE_HUMIDITY_CLUSTER, 0x0000)
-	} else if (isEWeLinkTh01()) {
-		return zigbee.readAttribute(0xFC45, 0x0000, ["mfgCode": 0x104E]) +   // New firmware
-			zigbee.readAttribute(0xFC45, 0x0000, ["mfgCode": 0xC2DF]) +   // Original firmware
-			zigbee.readAttribute(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000) +
+	} else if (isEWeLink()) {
+		return zigbee.readAttribute(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000) +
 			zigbee.readAttribute(0x0405, 0x0000) +
-			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020)
+			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021)
```
Comment (KKlimczukS) [inline devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy:184]: Why did You change that ? This attribute was previously supported by the model: TH01

File: devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy [hunk 2]
```diff
@@ -211,13 +210,11 @@ def configure() {
...
+	} else if (isEWeLink()) {
 		return refresh() +
-			zigbee.configureReporting(0xFC45, 0x0000, DataType.UINT16, 30, 3600, 100, ["mfgCode": 0x104E]) +   // New firmware
-			zigbee.configureReporting(0xFC45, 0x0000, DataType.UINT16, 30, 3600, 100, ["mfgCode": 0xC2DF]) +   // Original firmware
-			zigbee.batteryConfig() +
-			zigbee.temperatureConfig(3600, 7200) +
-			zigbee.configureReporting(0x0405, 0x0000, DataType.UINT16, 3600, 7200, null)
+			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 3600, 7200, 0x10) +
+			zigbee.temperatureConfig(10, 7200, 50) +
+			zigbee.configureReporting(0x0405, 0x0000, DataType.UINT16, 10, 7200, 300)
```
Comment (KKlimczukS) [inline devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy:217]: Why do You stop sending configuration for model Th01 ? Maybe You should split those configuration commands into 2 else-if sections ?

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy (hunk 1 above)
Comment (mingwei0827) [reply devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy:184]: We need to keep the same configuration, and the reported power percentage of our products will be more accurate.

File: devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy (hunk 2 above)
Comment (mingwei0827) [reply devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy:217]: I didn't stop sending the configuration for TH01, but the customer fed back some problems with the reported data, and we made some adjustments.

**Review by KKlimczukS — APPROVED** [review]
